src/main/java/me/juancarloscp52/entropy/client/integrations/discord/DiscordEventListener.java

```diff
@@ -58,41 +65,33 @@ public void onMessageReceived(@NotNull MessageReceivedEvent event) {
     @Override
     public void onMessageReactionAdd(@NotNull MessageReactionAddEvent event) {
         if(event.getMessageIdLong()==discordIntegration.lastId && event.getUserIdLong()!= discordIntegration.jda.getSelfUser().getIdLong()){
-            switch (event.getReactionEmote().getName()){
-                case "1️⃣":
-                    // Remove all previous user reactions to the message.
-                    event.retrieveMessage().complete().removeReaction("2️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("3️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("4️⃣",event.getUser()).queue();
-                    discordIntegration.votingClient.processVote(0, event.getUserId());
-                    break;
-                case "2️⃣":
-                    event.retrieveMessage().complete().removeReaction("1️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("3️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("4️⃣",event.getUser()).queue();
-                    discordIntegration.votingClient.processVote(1, event.getUserId());
-                    break;
-                case "3️⃣":
-                    event.retrieveMessage().complete().removeReaction("2️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("1️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("4️⃣",event.getUser()).queue();
-                    discordIntegration.votingClient.processVote(2, event.getUserId());
-                    break;
-                case "4️⃣":
-                    event.retrieveMessage().complete().removeReaction("2️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("3️⃣",event.getUser()).queue();
-                    event.retrieveMessage().complete().removeReaction("1️⃣",event.getUser()).queue();
-                    discordIntegration.votingClient.processVote(3, event.getUserId());
-                    break;
-                default: break;
+            List<UnicodeEmoji> emojisToRemove = new ArrayList<>();
+            UnicodeEmoji addedEmoji = event.getReaction().getEmoji().asUnicode();
+
+            emojisToRemove.add(Emoji.fromUnicode("1️⃣"));
+            emojisToRemove.add(Emoji.fromUnicode("2️⃣"));
+            emojisToRemove.add(Emoji.fromUnicode("3️⃣"));
+            emojisToRemove.add(Emoji.fromUnicode("4️⃣"));
```

Comment:
Could be written a bit more compactly:
```suggestion
            Stream.of("1️⃣", "2️⃣", "3️⃣", "4️⃣").map(Emoji::fromUnicode).forEach(emojisToRemove::add);
```